packages/react/runtime/__test__/typecheck/jsx-runtime.test-d.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+import { assertType, describe, test } from 'vitest';
+import { JSX } from '../../jsx-runtime/index';
+import React from 'react';
```

Comment:
```suggestion
```

Please add a `tsconfig.json` that extends the root `tsconfig.json` with `jsx` and `jsxImportSource`.